yarn.lock

```diff
@@ -9660,17 +8557,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/prettier@npm:^2.1.5":
-  version: 2.7.0
-  resolution: "@types/prettier@npm:2.7.0"
-  checksum: 5451430048c139456f14cc4eab8e1fd4d2dde4e0e10dbd3b49b8befa173f0958bf477575a4848bacfee5d42e46c4494dc9f5933fe8bcadf43b862741a7d049ad
-  languageName: node
```

Comment:
this is why we have an error, i wish eslint would warn us of this sort of thing.

Reply:
oh, nice catch, will update it tomorrowww

Reply:
I have a PR that solves this open FYI 😎

Reply:
which PR? 👀